confdb/aclocal_util.m4

```diff
@@ -1,3 +1,11 @@
+dnl PAC_ADD_CSV - Add value to comma separated list
+dnl Usage: PAC_ADD_CSV([foo], [bar])
+AC_DEFUN([PAC_ADD_CSV],[
+    AS_IF([test -n "$$2"],
+          [$2="$$2, $1"],
+          [$2=$1])
+])
```

Comment:
How about `PAC_APPEND_CSV` since we already use `PAC_APPEND_FLAG`?

Reply:
Done.